app/jenkins-for-jira-ui/src/api/fetchGlobalPageUrl.ts

```diff
@@ -1,5 +1,20 @@
 import { invoke } from '@forge/bridge';
 
+type FetchSiteNameProps = {
+	withProtocol: boolean
+};
+
+const fetchSiteName = async ({ withProtocol }: FetchSiteNameProps): Promise<string> => {
```

Comment:
/nit - as far as i can tell this is only called once in JenkinsSetup.ts passing withProtocol:false
 ....moving from nit to opinion piece now lol. 
 I would remove the flag arg entirely, it muddies the code and isn't adding any extra benefits, if the time comes that we need a protocolless we should make fetchSiteNameWithoutProtocol which could probably just leverage existingg....
 
```
 fetchSiteNameWithoutProtocol() {
  siteUrl = await fetchSiteUrl()
  protocolRegx = /^https?:\/\//
  return siteUrl.replace(protocolregex, '');
  }
```